lib/core.js

```diff
@@ -300,6 +300,15 @@ interface Iterable<T> {
     @@iterator(): Iterator<T>;
 }
 
+/* Generators */
+interface Generator<Y,R,N> {
+    @@iterator(): Iterator<Y>;
+    next(value?: N): IteratorResult<R|Y>;
```

Comment:
I half wonder if you're hitting the horrible Cache. Could you rebase past 7a4a6294b32d285c0b496570b911a07b00d31e44 and try again?